scripts/build-merge

```diff
@@ -122,7 +122,7 @@ echo-do mv doc/target/scala-2.11/unidoc stage/docs/scala/store/${version}
 # Create debian package
 STAGE_DEB=stage/pkgs/debian
 echo-do mkdir -p ${STAGE_DEB}
-./package.sh -t debian -V ${version} -j ${STAGE_FINAGLE}/finagle-server-${version}.jar -o ${STAGE_DEB}
+$DIR/scripts/package.sh -t debian -V ${version} -j ${STAGE_FINAGLE}/finagle-server-${version}.jar -o ${STAGE_DEB}
```

Comment:
Oops. Also, braces. ${DIR}. Finally, I think it should use `echo-do`, like all the other commands, so that the output of package.sh goes into the build.log rather than Jenkins, and so that we check the status and fail the build if necessary.